lib/diaspora_federation/entities/conversation.rb

```diff
@@ -0,0 +1,12 @@
+module DiasporaFederation
+  module Entities
+    class Conversation < Entity
+      property :guid
+      property :subject
+      property :created_at, default: -> { Time.now.utc }
+      entity :messages, [Entities::Message]
```

Comment:
It's been a while, but I think there only ever one message, but it has its own XML root element.
I reverse-enginieered the entities from the generated XML, so that *should* be correct...